specs/bin/crontabSpec.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var fs = require('fs');
+
+describe('crontab', function () {

```

Comment:
I think we should add a file to the repo like this: https://github.com/AnalyticalGraphicsInc/agi-cesium-cloud/blob/master/.gitattributes

rather than have a test.